### ISSUE TITLE
running node_modules/gulp/bin/gulp.js build throws error

### ISSUE THREAD

**Issue by vmahadev** [yeoman/generator-webapp#569]
node version : 6.4.0
eslintversion:v2.13.1

node_modules/gulp/bin/gulp.js build 
throws the following error
but node_modules/gulp/bin/gulp.js serve works fine

node_modules/eslint/lib/config/config-file.js:183
        throw e;
        ^

SyntaxError: Cannot read config file:
![screen shot 2016-09-15 at 3 52 23 am](https://cloud.githubusercontent.com/assets/1321782/18532394/6f5178f6-7af9-11e6-9574-9051b071aae6.png)

**Comment by kevva** [comment]
Seems like there's some syntax error in your `package.json`.

**Comment by vmahadev** [comment]
Thanks kevva here is my package.json

``` json
{
  "private": true,
  "engines": {
    "node": ">=4"
  },
  "devDependencies": {
    "babel-core": "^6.4.0",
    "babel-preset-es2015": "^6.3.13",
    "babel-register": "^6.5.2",
    "browser-sync": "^2.2.1",
    "del": "^1.1.1",
    "gulp": "^3.9.0",
    "gulp-autoprefixer": "^3.0.1",
    "gulp-babel": "^6.1.1",
    "gulp-cache": "^0.4.2",
    "gulp-cssnano": "^2.0.0",
    "gulp-eslint": "^2.0.0",
    "gulp-htmlmin": "^1.3.0",
    "gulp-if": "^2.0.0",
    "gulp-imagemin": "^2.2.1",
    "gulp-load-plugins": "^0.10.0",
    "gulp-plumber": "^1.0.1",
    "gulp-sass": "^2.0.0",
    "gulp-size": "^1.2.1",
    "gulp-sourcemaps": "^1.5.0",
    "gulp-uglify": "^1.1.0",
    "gulp-useref": "^3.0.0",
    "main-bower-files": "^2.5.0",
    "wiredep": "^2.2.2"
  },
  "eslintConfig": {
    "env": {
      "es6": true,
      "node": true,
      "browser": true
    },
    "rules": {
      "quotes": [
        2,
        "single"
      ]
    }
  }
}
```

**Comment by silvenon** [comment]
@vmahadev are you still having issues with this? The previous error was because of a syntax error, is there a different error now?

**Comment by dendrochronology** [comment]
Running `serve` works fine for me, but I can't get `build` to work at all. Tossing a couple log statements in reveals that uglify is processing my index.html as if it were JS in the `html` task, and it dies on the DOCTYPE line. This is with a fresh node LTS, yo, generator-webapp, blank project, etc. 

Shouldn't `$.if('*.js'` prevent that?

**Comment by silvenon** [comment]
@dendrochronology yeah, gulp-if has some problems with passing wrong files through. We added a workaround for this and I just released it in [v2.4.0](https://github.com/yeoman/generator-webapp/releases/tag/v2.4.0). The fix is 0ee21efe8ca883a7dcc6c4a64f151af5ad79c208, so please try it. 😉

I'm closing this due to inactivity. I will gladly reopen once we get new info.

**Comment by dendrochronology** [comment]
Yep, worked great. Thanks!
